app/assets/stylesheets/application.scss

```diff
@@ -0,0 +1,36 @@
+@import 'bootstrap-sprockets';
+@import 'bootstrap';
+
+/* universal */
+body {
+  padding-top: 60px;
+  }
+  section {
+  overflow: auto;
+  }
+  textarea {
+  resize: vertical;
```

Comment:
![image](https://user-images.githubusercontent.com/65896888/91004081-6576ea00-e5fd-11ea-90a1-01345523a10b.png)

Reply:
done

Reply:
please check it again